### ISSUE TITLE
Is it possible to add error detection to SMS send

### ISSUE THREAD

**Issue by ndastur** [cjlotz/Xamarin.Plugins#75]
https://developer.android.com/reference/android/telephony/SmsManager.html#sendTextMessage(java.lang.String, java.lang.String, java.lang.String, android.app.PendingIntent, android.app.PendingIntent)

It looks like there can be an Intent to receive the status of a sent SMS.

**Comment by soroshsabz** [comment]
Related to PR #115 and #114

**Comment by soroshsabz** [comment]
@VahidShir please check this API, and check can you use this in `OnSmsDeliveredResult` or not?

**Comment by VahidShir** [comment]
@soroshsabz  Yes. The PR #115 works, and it returns the result, i.e whether the SMS was delivered successfully or not.
